html/changelogs/example.yml

```diff
@@ -27,7 +27,7 @@
 #################################
 
 # Your name.
-author: ChangeMe
+author: SueTheCake
```

Comment:
You accidentally edited the example changelog instead of making a new one here.

Reply:
augh im stupid